src/main/java/service/BlackjackService.java

```diff
@@ -0,0 +1,81 @@
+package service;
+
+import domain.PlayerGameResult;
+import domain.card.Deck;
+import domain.participant.Participant;
+import domain.participant.Participants;
+
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class BlackjackService {
+
+    private static final String HIT = "y";
+    private static final String STAND = "n";
+    private static final String ERROR_INVALID_COMMAND = "[ERROR] y 혹은 n만 입력하실 수 있습니다.";
+
+    private final Deck deck;
+    private final Participants participants;
+
+    private BlackjackService(Deck deck, Participants participants) {
+        this.deck = deck;
+        this.participants = participants;
+    }
+
+    public static BlackjackService of(List<String> playersName) {
+        return new BlackjackService(Deck.create(), Participants.of(playersName));
+    }
+
+    public void start() {
+        participants.initHand(deck);
+    }
+
+    public boolean existNextPlayerTurn() {
+        return participants.getNextTurnPlayer().isPresent();
+    }
+
+    public Participant getNextPlayer() {
+        return participants.getNextTurnPlayer().get();
+    }
+
+    public void nextTurn(String hit) {
+        Participant nextPlayer = participants.getNextTurnPlayer().get();
+
+        if (hit.equals(STAND)) {
```

Comment:
equals 비교는 null일 수 없는 것을 앞에 두고 비교하는게 좋아요. 객체를 비교하는 경우 null인 객체가 들어온다면 equals 호출시 NPE가 발생할 수 있기 때문이에요. 다른 부분도 해당 내용을 수정해보면 좋을 듯 싶어요 :)
```suggestion
        if (STAND.equals(hit)) {
```

Reply:
이건 언급해주시 않으셨으면 계속 모르고 프로그래밍 했을 것 같습니다. 감사합니다!